java/example/calypso/android/omapi/src/main/kotlin/org/eclipse/keyple/example/calypso/android/omapi/activity/CoreExamplesActivity.kt

```diff
@@ -108,25 +110,34 @@ class CoreExamplesActivity : ExamplesActivity() {
         val seAidPrefix = "A000000404012509"
 
         /* AID based selection (1st selection, later indexed 0) */
-        seSelection.prepareSelection(GenericSeSelectionRequest(SeSelector(
-                SeCommonProtocols.PROTOCOL_ISO7816_3, null,
-                SeSelector.AidSelector(SeSelector.AidSelector.IsoAid(seAidPrefix),
-                        SeSelector.AidSelector.FileOccurrence.FIRST,
-                        SeSelector.AidSelector.FileControlInformation.FCI))))
+        seSelection.prepareSelection(GenericSeSelectionRequest(
+                SeSelector.builder()
+                        .seProtocol(SeCommonProtocols.PROTOCOL_ISO7816_3)
+                        .aidSelector(SeSelector.AidSelector.builder()
+                                .aidToSelect(seAidPrefix)
+                                .fileOccurrence(SeSelector.AidSelector.FileOccurrence.FIRST)
+                                .fileControlInformation(SeSelector.AidSelector.FileControlInformation.FCI).build())
```

Comment:
Add the use of static import to clarify the usage of SeSelector & PoSelector sub-classes